catalogue/webapp/pages/search/index.tsx

```diff
@@ -198,28 +192,26 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     const defaultProps = removeUndefinedProps({
       serverData,
       query,
-
-      // TODO Harrison to explore what properties we'd want here
       pageview: {
         name: 'search',
         properties: {},
       },
     });
 
-    // Stop here if no query has been entered
-    if (!params.query) {
-      return {
-        props: defaultProps,
-      };
-    }
-
     try {
       // Stories
+      // Setting a default order of descending publication date as default state
```

Comment:
This comment tells me what the code does, but it'd be more useful to say why it does that. Why did we pick this as the default?

Reply:
I thought it was the most logical, Jenn then agreed with it in a Slack thread a few days after I offered this PR for review. I can tweak the comment.